### ISSUE TITLE
problem doing a full library update

### ISSUE THREAD

**Issue by Butterfly-Dragon** [openaudible/openaudible#371]
![image](https://user-images.githubusercontent.com/33538906/82615424-9f81bb80-9bca-11ea-9018-a4103d0d2c90.png)

my guess is... that it did not understand where my library ends.😅

at 20 titles per page there are 72 pages to browse in my library... where did it get *looks up where it's at right now...* 1713 pages?!?

**Comment by iamawo** [comment]
Same problem on the Mac version - for Full or Quick sync
![Screen Shot 2020-05-22 at 17 28 10](https://user-images.githubusercontent.com/65780351/82689081-d64ee480-9c51-11ea-971d-003aa6fa3c1e.png)

**Comment by Etumos** [comment]
Same issue on the Linux version. It scanned like 20k plus books.

**Comment by lupuslazuli** [comment]
Same issue on Linux and Windows. Seams like it's processing the first Page of the libary over and over again.

**Comment by matschundbrei** [comment]
Most probably there's something changed in the 'last-page'  notification/mechanism on the audible site.
I tried manually changing the '&page=' param on the page and indeed could change it to any number and got the last page of my library (which is page 3) displayed.

This might also be due to the fact, that the 'NextPage' button is still displayed, but disabled through html.

Note that in the below example I the disabled 'next' button still contains a valid link to page 4 (which does not exist and will just display page 3)

To me the following renders in this XPATH: /html/body/div[3]/div[8]/div[7]/form/div/div/div[2]/div/span/ul/li[5]/span or the following css-selector respectively: .nextButton

(Whitespaces and user data removed)
```html
<span class="bc-button
  bc-button-secondary
  nextButton  
  
  
  bc-button-small 
  bc-button-inline 
  bc-button-disabled">
      <a class="bc-button-text" disabled="true" aria-disabled="true" data-url="/library/titles?ref=a_library_t_c6_pageNext&amp;pf_rd_p=REDACTED&amp;pf_rd_r=REDACTED" data-name="page" data-value="4" href="/library/titles?pf_rd_p=REDACTED&amp;pf_rd_r=REDACTED&amp;ref=a_library_t_c6_pageNext&amp;sortBy=PURCHASE_DATE.dsc&amp;page=4" tabindex="0">
        <span class="bc-text
    bc-button-text-inner 
    bc-size-small">

<i aria-hidden="true" class="bc-icon
    
		bc-icon-chevron-right">
</i>
<span class="bc-text
    bc-pub-offscreen">Eine Seite vorwärts</span>
  </span>
      </a>
    </span>
```

**Comment by openaudible** [comment]
I think so too.. and the download link changes randomly each time the "last page" is loaded, causing a "change" in the book data. I'll have a fix for next release. Thanks all for the help.

**Comment by openaudible** [comment]
Fixed in 2.0.9.
